### PR TITLE
Chinese language patch description change and Tatarigoroshi link error

### PR DESCRIPTION
**Pull request by CN-ycx**

### CODE REVIEW

**Comment by drojf** [comment]
The link https://higurashi.ycx-studios.site/guide/07th-mod/main/ gives me a 404, are you sure it is correct?

image

File: docs/Snippets/third-party-features.md [hunk 1]
```diff
@@ -32,26 +32,28 @@ These features can be installed as an installer option (or manually), but please
...
+    An addon which provides a conversion of Ch.1 to Ch.6 (so far) to Chinese. [More details in Chinese here](https://higurashi.ycx-studios.site).
 
     Please note:
 
     * 07th-mod does not directly support this patch - instead:
 
         * Visit the [patch list page](https://higurashi.ycx-studios.site/patch/list/) for all available patches
         * Contact them directly on [their website](https://higurashi.ycx-studios.site/about/)
-        * A version in Simplified Chinese of the [install guide](https://higurashi.ycx-studios.site/guide/07th-mod/main/) is available in their website
+        * A version in Chinese of the [install guide](https://higurashi.ycx-studios.site/guide/07th-mod/main/) is available in their website
```
Comment (drojf) [inline docs/Snippets/third-party-features.md]: This link https://higurashi.ycx-studios.site/guide/07th-mod/main/ gives me a 404 error

File: docs/Snippets/third-party-features.md [hunk 2]
```diff
@@ -32,26 +32,28 @@ These features can be installed as an installer option (or manually), but please
...
+    非官方汉化补丁, 目前仅提供第一章到第六章的版本. [更多信息请点这里](https://higurashi.ycx-studios.site).
     
     请注意:
     
     * 07th-mod 并不直接为该补丁提供支持 - 但你可以:
     
         * 访问[汉化补丁列表](https://higurashi.ycx-studios.site/patch/list/)查看所有可用的补丁
         * 直接在[补丁发布页面](https://higurashi.ycx-studios.site/about/)联系我们
-        * 另外我们也提供[07th-mod 使用指南](https://higurashi.ycx-studios.site/guide/07th-mod/main/)的简体中文翻译版本
+        * 我们提供[07th-mod 使用指南](https://higurashi.ycx-studios.site/guide/07th-mod/main/)的中文翻译版本
```
Comment (drojf) [inline docs/Snippets/third-party-features.md]: This link https://higurashi.ycx-studios.site/guide/07th-mod/main/ gives me a 404 error

File: docs/Snippets/third-party-features.md (hunk 1 above)
Comment (CN-ycx) [reply docs/Snippets/third-party-features.md]: fix, sorry XD

File: docs/Snippets/third-party-features.md (hunk 2 above)
Comment (CN-ycx) [reply docs/Snippets/third-party-features.md]: fix, sorry XD

**Comment by CN-ycx** [comment]
You can check the new link now, and I add the fast download site link provide by us, thanks!

**Comment by drojf** [comment]
@CN-ycx Out of curiosity, how slow is downloading from Github where you are? I noticed you said you rehost the files because the download speed is slow?

**Comment by drojf** [comment]
oh, was it actually 07th-mod.com which was slow (before we migrated?)

I was using Google Translate to read your wiki so I may have misunderstood

**Comment by CN-ycx** [comment]
@drojf Thank you for being so interested.
In China Mainland, all websites will be government censored by [Great Firewall](https://en.wikipedia.org/wiki/Great_Firewall), if some website seems illegal, then usually slow down internet traffic or fully block by GFW. 
Websites located overseas have particularly slow access speeds at night (corresponding to daytime in the United States), including Github and Cloudflare.
Similarly, Github and Cloudflare in some regions of China may limit or block by GFW, because many people abuse these services to host projects that violate laws.
For our, there are many CDN services provided by programmers, but many game players blame the slow download speed because they do not use these services frequently and do not understand this mechanism.

In China, we usually choose to store our files on online drives. Such as Baidu Netdisk, just like Mega and Google Drive.
These online storage services are also VIP based, and without buying VIP only speed limited downloads can be made.
Therefore, we have chosen to build a download site to provide players with a full speed download experience, which is currently available for free.

Finally, thanks so much for creating these patches for Higurashi players. If there is anything I can do to help, I am willing to do my best to assist!

**Pull request merged**